src/engine/renderer/tr_local.h

```diff
@@ -3383,7 +3382,19 @@ inline bool checkGLErrors()
 	void     GLimp_SyncRenderThread();
 	void     GLimp_WakeRenderer( void *data );
 
-	void     GLimp_LogComment( const char *comment );
+inline bool GLimp_isLogging() {
+	return r_logFile.Get() && GLEW_ARB_debug_output;
+}
+
+// Never use GLimp_LogComment() directly, use the GLIMP_LOGCOMMENT() wrapper instead.
+void GLimp_LogComment( std::string comment, bool wrapped = false );
```

Comment:
`bool wrapped` seems like overkill. Just give the function a name hinting it is not intending to be used normally, like with the trailing underscore on `GL_CheckErrors_`

Reply:
Done.